Updated focus descriptions

christinem opened the pull request:
I was looking at the focus descriptions in the course calendar and noticed alot of them were updated, so this PR reflects the changes.

david-yz-liu commented on public/js/graph/sidebar/focus_descriptions.js:
```diff
@@ -7,9 +7,11 @@ var HCIDescription = 'Human-Computer Interaction (HCI) is the scientific study o
                      '<em>[1.]</em> CSC300H1, CSC301H1, CSC318H1, CSC428H1 <br>' +
-                     '<em>[2.]</em> SOC101Y1, SOC200H1, SOC202H1, SOC302H1 <br>' + 
+                     '<em>[2.]</em> SOC101Y1, SOC200H1, SOC202H1, SOC302H1  [To enrol in restricted SOC courses, ' +
+                     'please contact the CS Undergraduate Office in the July preceeding the academic year in which you plan to take the course] <br>' + 
```
`preceeding` should be `preceding`

david-yz-liu commented on public/js/graph/sidebar/focus_descriptions.js:
```diff
@@ -31,7 +33,7 @@ var sciDescription = 'Scientific computing studies the world around us. Known an
                      'Required Courses: <br><br>' +
                      '<em>[1.]</em> MAT235Y1/MAT237Y1/MAT257Y1 <br>' +
-                     '<em>[2.]</em> 1.5 FCE from the following: CSC336H1, CSC350H1, CSC351H1, CSC446H1, 456H1 <br>' +
+                     '<em>[2.]</em> 1.5 FCE from the following: CSC336H1, CSC346H1, CSC446H1, CSC456H1, CSC466H1 <br>' +
```
CSC346 isn't a course - perhaps CSC436?

david-yz-liu commented on public/js/graph/sidebar/focus_descriptions.js:
```diff
@@ -157,15 +158,15 @@ var theoryDescription = 'Why is it easy to sort a list of numbers, but hard to b
-                        'CSC322H5/MAT302H5, CSC422H5; An appropriate CSC494H1/CSC495 project supervised by a faculty member ' +
-                        'from the Theory group or a relevant introductory graduate course in theory may also be used to ' +
-                        'satisfy this requirement.  (Note that students must petition to take a graduate course.) <br>' +
+                        'At UTM: CSC322H5/MAT302H5, CSC422H5; CSC494H1/CSC495 project supervised by a faculty member ' +
```
Seems `"At UTM"` is repeated

david-yz-liu commented:
@christinem Thanks for doing this! I noticed a couple of typos, if you could please fix them.

christinem commented:
Thanks @david-yz-liu, I made the fixes! Plus, I realized the focus arrays (that determine which courses are highlighted per focus) also needed to be changed to accommodate the courses added, removed, and moved to "Suggested" instead of "Required", so I've added that as well.

david-yz-liu commented:
Thanks, @christinem!
